tests/local/e2e_test.go

```diff
@@ -89,6 +95,8 @@ var _ = ginkgo.BeforeSuite(func() {
 		LocalNetworkInstance.Dir(),
 		ginkgo.ReportEntryVisibilityFailureOrVerbose,
 	)
+
+	log.Info("Set up ginkgo before suite")
```

Comment:
I'd prefer a message like "Finished ginkgo before suite set up" because "Set up" could also be interpreted as an imperative mood narration of what's about to happen rather than a past tense statement of what has already happened.